src/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobBuilder.cs

```diff
@@ -305,7 +305,7 @@ public byte[] ToArray(int start, int byteCount)
                         break;
                     }
 
-                    Buffer.BlockCopy(chunk._buffer, Math.Max(start - chunkStartPosition, 0), result, resultOffset, bytesToCopy);
+                    Array.Copy(chunk._buffer, Math.Max(start - chunkStartPosition, 0), result, resultOffset, bytesToCopy);
```

Comment:
Curious what the issue was here? Just style preference for Array.Copy for byte[]?